segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -192,6 +202,20 @@ private void initializeZookeeper() throws Exception {
         zkc.create(znodePath.toString(), new byte[0], ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
     }
 
+    private String loadPasswdFromFile(String tLSKeyStorePasswordPath) {
```

Comment:
Yet another instance of this exact method here. Please create a shared method or use `FileUtils.readFileToString`

Reply:
This version of the method is not deprecated:

https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#readFileToString(java.io.File,%20java.nio.charset.Charset)

what version of commons are we actually using?

Reply:
Please change `passwd` to `password` everywhere in this code

Reply:
And please remove this method and use the shared one

Reply:
Fixed. Using the non deprecated method through a shared version.